helm/cluster/templates/clusterapi/controlplane/_helpers_postkubeadmcommands.tpl

```diff
@@ -0,0 +1,3 @@
+{{- define "cluster.internal.controlPlane.kubeadm.postKubeadmCommands" }}
```

Comment:
Helper templates for post-kubeadm commands in KubeadmControlPlane (it just includes common commands here, but a later PR adds CP-specific commands as well).